develop/scan-images/index.md

```diff
@@ -5,12 +5,13 @@ keywords: docker scan, scan, images, snyk, vulnerability
 ---
 
 {% include sign-up-cta.html
-  body="You can now get 10 free scans per month as part of your Docker subscription. Sign in to Docker to start scanning your images for vulnerabilities."
-  header-text="This feature requires a Docker subscription"
+  body="Did you know that you can now get 10 free scans per month? Sign in to Docker to start scanning your images for vulnerabilities."
+  header-text="Scan your images for free"
   target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_scan"
 %}
 
-This page contains recommendations and best practices for scanning and building secure images.
+This page contains recommendations and best practices for scanning and building
+secure images.
```

Comment:
There're a few places in this PR where sentences have been broken across multiple lines like this. Was that deliberate?

Reply:
The IDE wraps it to 80 chars to help with review, etc

Reply:
I guess this is personal preference, but I think that makes reviews harder. E.g. suppose a paragraph is divided into six lines. If you add something to the first line, the IDE will reflow all the lines below it, and the diff will just show six totally modified lines.

Conversely, if it's kept as one line, GitHub shows nice highlighting of precisely which parts have changed in the PR:

![image](https://user-images.githubusercontent.com/70579116/146083639-a0c07cec-50c8-4f79-8a5f-0aaf926e298c.png)

Reply:
The reverse usually is true as well; multiple reviews and/or multiple pull request touching the same file are more likely to interact badly if it's "one line per paragraph", which is why we generally stick to hard-wrapping.